specification/schemas/Shipment.json

```diff
@@ -255,6 +255,30 @@
                 }
               ]
             },
+            "shipment_surcharges": {
+              "allOf": [
+                {
+                  "$ref": "#/components/schemas/ShipmentSurchargeRelationship"
```

Comment:
The `ShipmentSurchargeRelationship` schema describes a single resource with its own `data` attribute.
We need to also create a ShipmentSurcharge**s**Relationship that has a single `data` property, with multiple `MinimalShipmentSurcharge` schemas (just `type` and `id`).